static/js/util/util.js

```diff
@@ -227,44 +228,72 @@ complete profile consists of:
     `currently employed == 'yes'`
 */
 export function validateProfileComplete(profile) {
```

Comment:
(previously we were just checking that the returned `errors` objects was non-empty, so we didn't care about the specific messages, but now we want them to dispatch to the store and so on)